src/nikhilMJun20/FibonacciN.java

```diff
@@ -0,0 +1,42 @@
+/*To print first n Fibonacci Numbers.
+Input n = 5
+Output = 1 1 2 3 5 8
+
+Input: n = 10
+Output : 1 1 2 3 5 8 13 21 34 55
+
+Hint :
+The next number is found by adding up the two numbers before it:
+the 2 is found by adding the two numbers before it (1+1),
+the 3 is found by adding the two numbers before it (1+2),
+the 5 is (2+3),
+and so on!
+*/
+
+package nikhilMJun20;
+
+import java.util.Scanner;
+
+public class FibonacciN {
+	void displayFibonacci(int order) {
+		System.out.println("\nFibonacci Series of Order "+order+":-\n");
+		int num1=1,num2=1,num3=0;
+		for(int index=0; index<=order; index++) {
+			if(index<2)
```

Comment:
--You can assign 0 to num1 variable and then there is no need for this if else logic so you can reduce some lines of code.

Reply:
->Sure. I can make that change which is more efficient.
But this requires additional print statement outside the loop - which is kind of like an adjustment made by us without implementing logic of Fibonacci series. Logic should be like unless there are 2 elements which we can add, we should not proceed to 3rd element. 
Let me know your thoughts on this.

Reply:
-- Well if logically thinking this makes sense.
--However if we implement suggested logic, code can be little efficient.
-- I am also not sure which approach is better, so let's our reviewer take the call.

Reply:
void printFibonacciNumber(int number) {
		int num =0; 
		int num1 = 0, num2=1;
		for(int index=1; index<=5; index++) {
			System.out.print(num2+" ");
			num = num1 + num2;
			num1= num2;
		    num2 = num;
		}
	}
Nikhil if block is not required.